mavlink/share/pyshared/pymavlink/examples/magfit_delta.py

```diff
@@ -4,6 +4,7 @@
 fit best estimate of magnetometer offsets using the algorithm from
 Bill Premerlani
 '''
+from __future__ import print_function
```

Comment:
But it doesn't make sense to fix pymavlink in here because that's usually hosted elsewhere: https://github.com/ArduPilot/pymavlink

Reply:
If you pip install a dependency then you can ignore it’s syntax errors but if you vendor-in code then you own it’s syntax errors.  Last time I looked, pymavlink had the same flake8 tests.

Reply:
Yes... https://github.com/ArduPilot/pymavlink/blob/master/.travis.yml#L19

Reply:
> if you vendor-in code then you own it’s syntax errors

That's unfortunately true.

Reply:
You have an OLD copy of upstream.  Please refresh.

Reply:
It's gone now, you can rebase or merge.